pp/integrations/plugin-packs/procedures/cloud-azure-management-insightsmetrics.md

```diff
@@ -5,182 +5,496 @@ title: Azure InsightsMetrics
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-
 ## Overview
 
-> This Pack is in an experimental state (version 2.x.X).
-
 The Centreon Pack *Azure InsightsMetrics* aims to monitor additional indicators that are not available on the Azure Monitor feature.
```

Comment:
```suggestion
The Centreon **Azure InsightsMetrics** connector aims to monitor additional indicators that are not available on the Azure Monitor feature.
```

Reply:
what do you mean by " additional indicators that are not available on the Azure Monitor feature."?